aptos-move/framework/aptos-framework/sources/resource_account.spec.move

```diff
@@ -109,6 +115,7 @@ spec aptos_framework::resource_account {
         aborts_if !simple_map::spec_contains_key(container.store, resource_addr);
         aborts_if !exists<account::Account>(resource_addr);
         ensures simple_map::spec_contains_key(old(global<Container>(source_addr)).store, resource_addr) &&
-        simple_map::spec_len(old(global<Container>(source_addr)).store) == 1 ==> !exists<Container>(source_addr);
+            simple_map::spec_len(old(global<Container>(source_addr)).store) == 1 ==> !exists<Container>(source_addr);
+        ensures !simple_map::spec_contains_key(global<Container>(source_addr).store, resource_addr);
```

Comment:
@UIZorrot Strictly speaking, this is a little be misleading because Container may no longer exist if store is empty.

Reply:
`ensures exists<Container>(source_addr) ==> !simple_map::spec_contains_key(global<Container>(source_addr).store, resource_addr);`?